deeposlandia/dataset.py

```diff
@@ -256,8 +264,12 @@ def _preprocess(self, image_filename, output_dir, aggregate, labelling=True):
             if aggregate:
                 final_img_out = self.group_image_label(final_img_out)
 
-            labels = utils.mapillary_label_building(final_img_out, self.label_ids)
-            new_out_filename = os.path.join(output_dir, 'labels', label_filename.split('/')[-1])
+            labels = utils.mapillary_label_building(final_img_out,
+                                                    self.label_ids)
+            new_out_filename = os.path.join(output_dir, 'labels',
+                                            label_filename.split('/')[-1])
```

Comment:
maybe `os.path.basename` can be used instead of `split('/')`. For instance, in Windows, this does not work.

Reply:
Yes, good point, I did not know the `basename` function, and missed the doc. I change that.